.github/workflows/ci-cd.yml

```
@@ -119,54 +142,79 @@
+          environments: ${{ env.PIXI_ENV }}
+          pixi-version: v0.33.0
+          cache: true
+          locked: false # wont be the same because of the tag
+
+      - name: Publish to TestPyPI
+        env:
+          HATCH_INDEX_USER: __token__
+          HATCH_INDEX_AUTH: ${{ secrets.TEST_PYPI_API_TOKEN }}
+        run: |
+          pixi run -e ${{ env.PIXI_ENV }} publish-test
+  
```

yamllint

[error] trailing spaces

(trailing-spaces)